CausalEstimate/simulation/binary_simulation.py

```
@@ -1,6 +1,12 @@
 import numpy as np
 import pandas as pd
 from scipy.special import expit as logistic
+from CausalEstimate.utils.constants import (
+    TREATMENT_COL,
+    OUTCOME_COL,
+    PID_COL,
```

Ruff (0.8.2)

`CausalEstimate.utils.constants.PID_COL` imported but unused

Remove unused import: `CausalEstimate.utils.constants.PID_COL`

(F401)